cartoframes/data/enrichment/enrichment_service.py

```diff
@@ -162,14 +170,6 @@ def __process_agg_operators(agg_operators, variables):
     return agg_operators_result
```

Comment:
`__process_agg_operators` method should also take into account what happens if this argument is a string, as we're doing in the `enrich_polygons` method. If it's a string, it returns `'str' object has no attribute 'copy'`

Reply:
Fixed!